src/grading-settings/data/api.js

```diff
@@ -2,10 +2,11 @@
 import { camelCaseObject, getConfig } from '@edx/frontend-platform';
 import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 
-import { convertObjectToSnakeCase } from '../../utils';
+import { deepConvertingKeysToSnakeCase } from '../../utils';
 
 const getApiBaseUrl = () => getConfig().STUDIO_BASE_URL;
 export const getGradingSettingsApiUrl = (courseId) => `${getApiBaseUrl()}/api/contentstore/v1/course_grading/${courseId}`;
+export const getCourseSettingsApiUrl = (courseId) => `${getApiBaseUrl()}/api/contentstore/v1/course_settings/${courseId}`;
```

Comment:
can you use selector from api `getCourseSettingsApiUrl` on the module schedule&details instead?

Reply:
Discussed on call